electricitymap/contrib/parsers/tests/__snapshots__/test_ENTSOE.ambr

```diff
@@ -5873,779 +5873,9255 @@
+      'price': 78.0,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 14, 0, tzinfo=datetime.timezone.utc),
+      'price': 84.01,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 15, 0, tzinfo=datetime.timezone.utc),
+      'price': 97.05,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 16, 0, tzinfo=datetime.timezone.utc),
+      'price': 106.5,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 17, 0, tzinfo=datetime.timezone.utc),
+      'price': 112.82,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 18, 0, tzinfo=datetime.timezone.utc),
+      'price': 122.76,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 19, 0, tzinfo=datetime.timezone.utc),
+      'price': 121.99,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 20, 0, tzinfo=datetime.timezone.utc),
+      'price': 114.71,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2023, 5, 8, 21, 0, tzinfo=datetime.timezone.utc),
+      'price': 104.04,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'DK-BHM',
+    }),
+  ])
+# ---
+# name: test_fetch_prices_intraday
+  list([
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2025, 9, 28, 22, 0, tzinfo=datetime.timezone.utc),
+      'price': 75.0,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'ES',
+    }),
+    dict({
+      'currency': 'EUR',
+      'datetime': datetime.datetime(2025, 9, 28, 22, 0, tzinfo=datetime.timezone.utc),
+      'price': 77.77,
+      'source': 'entsoe.eu',
+      'sourceType': <EventSourceType.measured: 'measured'>,
+      'zoneKey': 'ES',
+    }),
```

Comment:
Here are the tricky datapoints...

We need to figure out what to do with these and what the forecast team actually needs.